ops/terraform/services/base/values/prod.yaml

```
@@ -172,7 +172,7 @@
 /bfd/${env}/pipeline/nonsensitive/rda/job/starting_fiss_seq_num: UNDEFINED
 /bfd/${env}/pipeline/nonsensitive/rda/job/starting_mcs_seq_num: UNDEFINED
 /bfd/${env}/pipeline/nonsensitive/rda/new_relic/app_name: BFD Pipeline (${env})
-/bfd/${env}/pipeline/nonsensitive/rda/cleanup/enabled: "true"
+/bfd/${env}/pipeline/nonsensitive/rda/cleanup/enabled: "false"
```

We can leave the cleanup job on since we have the database copy already.
```suggestion
/bfd/${env}/pipeline/nonsensitive/rda/cleanup/enabled: "true"
```